### PR TITLE
Mention new templates.xml memory settings in troubleshooting section

### PR DESCRIPTION
**Pull request by sbesson**
Fixes https://trac.openmicroscopy.org.uk/ome/ticket/12527

### CODE REVIEW

File: omero/sysadmins/troubleshooting.txt [hunk 1]
```diff
@@ -121,7 +121,11 @@ Server fails to start
 5. If you see an error message mentioning "Protocol family unavailable",
    you will need to set the :property:`Ice.IPv6` property with
    :omerocmd:`config set IPv6 0`.
-6. If OMERO starts up, but fails to respond to connection requests, check
+6. If you upgraded from a 5.0.2 server or older and copied the entire content
+   of the :file:`etc/grid` directory from the old server to the new server,
+   you will need to revert the changes made :file:`templates.xml` to
```
Comment (mtbc) [inline omero/sysadmins/troubleshooting.txt]: "made to"

**Comment by ximenesuk** [comment]
Final change looks good, any further review needed?

**Comment by hflynn** [comment]
Is this ready to merge @sbesson ?

**Comment by sbesson** [comment]
Assuming you are happy with it, I am not planning any additional change.

**Comment by hflynn** [comment]
Looks fine to me, merging.

**Comment by hflynn** [comment]
Does this need rebased?

**Comment by sbesson** [comment]
--rebased-to #1107

**Pull request merged**